specification/billing/resource-manager/Microsoft.Billing/stable/2021-10-01/payment.json

```diff
@@ -0,0 +1,822 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-10-01",
+    "title": "PaymentManagementClient",
+    "description": "Payment management client provides access to payment resources for Azure."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Billing/paymentMethods": {
```

Comment:
>"/providers/Microsoft.Billing/paymentMethods": [](http://example.com/codeflow?start=4&length=46)

I see no PUT. How will it be created?

Reply:
There are compliance requirements around adding or editing a credit card into Microsoft, which are not possible to be fulfilled through public APIs at this point. For example, European customers have to go through PSD2 challenge, which basically means user has to do MFA authentication with bank before using the credit card. This can only be done through UIs, but not through APIs. So users need to go to portal.azure.com to add or change credit cards.